src/main/java/org/folio/bulkops/processor/MarcInstanceDataProcessor.java

```diff
@@ -53,6 +54,8 @@ private void applyRuleToRecord(BulkOperationMarcRule rule, Record marcRecord) th
         case REPLACE_WITH -> processFindAndReplace(rule, marcRecord);
         default -> throw new BulkOperationException(String.format("Action FIND + %s is not supported yet.", actions.get(1).getName()));
       }
+    } else if (REMOVE_ALL.equals(actions.get(0).getName())) {
```

Comment:
Hi @obozhko-folio , looks here is equals checking for enums, it is better to use == for them .